### PR TITLE
add initializationTime and platform to PINWHEEL_INIT payload

### PR DESCRIPTION
**Pull request by paveu-piotrowski**

### CODE REVIEW

File: pinwheel-android/src/main/java/com/underdog_tech/pinwheel_android/Pinwheel.kt [hunk 1]
```diff
@@ -10,21 +10,25 @@ object Pinwheel {
...
 
         val pinwheelJSInterface = PinwheelJavaScriptInterface(callback)
         pinwheelJSInterface.bind((webView))
 
         webView.loadUrl(CDN_URL)
     }
+
+    private fun getUnixTimestamp(): Long {
+        return System.currentTimeMillis() / 1000L
+    }
```
Comment (roscioli) [inline pinwheel-android/src/main/java/com/underdog_tech/pinwheel_android/Pinwheel.kt:33]: Is the timestamp returned from `getUnixTimestamp` in milliseconds?

Comment (paveu-piotrowski) [reply pinwheel-android/src/main/java/com/underdog_tech/pinwheel_android/Pinwheel.kt:33]: Currently in seconds. Will fix in a minute.

**Comment by roscioli** [comment]
Let's wait until https://github.com/underdog-tech/link/pull/421 is merged in because then we can test this live with the production link modal bundle

**Comment by roscioli** [comment]
Looks good. We just have to QA this before moving on. Since we have no staging env or deployment for this, let's just test it out on your end and then we can check the datadog logs with the token ID you used in your session (be sure to save it, i can check on datadog for it).

**Pull request merged**